### DISCUSSION TITLE
Windows server 2019?

### DISCUSSION THREAD

**Discussion by robba86** [simeononsecurity/Windows-Optimize-Harden-Debloat#37] [General]
What do i need to make it work for this distro? It's kind a like win 10 ent? I May be wrong, but i like your opinion on it 😄

[a comment by an automated account is not shown]

**Comment by simeononsecurity** [comment]
> What do i need to make it work for this distro? It's kind a like win 10 ent? I May be wrong, but i like your opinion on it 😄

Server OS'es don't really come with a lot of bloat.
What are your main goals?

**Comment by simeononsecurity** [comment]
Also this is not a place for discussions and questions. Please see https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/discussions

**Comment by simeononsecurity** [comment]
@robba86
